### PR TITLE
Techweb fixes

### PR DESCRIPTION
**Pull request by silicons**
Fixes efficiency memes and some things being places they shouldn't be printable from.

### CODE REVIEW

**Comment by SpaceManiac** [comment]
While you're at it, I think the cargo express console board is showing up in the protolathe rather than circuit imprinter.

**Comment by kevinz000** [comment]
path please

**Comment by SpaceManiac** [comment]
`/obj/item/circuitboard/computer/cargo/express`

**Comment by kevinz000** [comment]
done

File: code/modules/research/designs/comp_board_designs.dm [hunk 1]
```diff
@@ -41,118 +44,135 @@
...
 	category = list("Computer Boards")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
 
 /datum/design/board/operating
 	name = "Computer Design (Operating Computer)"
 	desc = "Allows for the construction of circuit boards used to build an operating computer console."
 	id = "operating"
 	build_path = /obj/item/circuitboard/computer/operating
 	category = list("Computer Boards")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
```
Comment (Wilchenx) [inline code/modules/research/designs/comp_board_designs.dm]: robotics totally doesn't need this :^)

**Comment by davethwave** [comment]
@SpaceManiac cargo doesn't start with a circuit printer.

**Comment by SpaceManiac** [comment]
They have a techfab (dual protolathe + circuit printer) now, I thought?

File: code/modules/research/designs/comp_board_designs.dm (hunk 1 above)
Comment (kevinz000) [reply code/modules/research/designs/comp_board_designs.dm]: woops

**Comment by kevinz000** [comment]
spacemaniac is correct

**Pull request merged**